### PR TITLE
Merge final version of P-I branch

### PR DESCRIPTION
**Pull request by iarinamd**

### CODE REVIEW

**Comment by 2560p** [comment]
@ZsomborHajzer, the pull request proposes to merge the final part as well as the intermediate ones. 

There is going to be a need of removing those on the `main-developement` branch.

**Review by ZsomborHajzer — APPROVED** [review]
Looks good so far

File: CAR P-I/actual/actual.ino
Comment (2560p) [inline CAR P-I/actual/actual.ino:1]: LGTM, will be removed from the final version, so the commented code will not get it to the branch.

File: CAR P-I/basic_movement_v1/basic_movement_v1.ino
Comment (2560p) [inline CAR P-I/basic_movement_v1/basic_movement_v1.ino:1]: LGTM, will be removed from the final version, so the commented code will not get it to the branch.

File: CAR P-I/better_turns/better_turns.ino
Comment (2560p) [inline CAR P-I/better_turns/better_turns.ino:1]: LGTM, will be removed from the final version, so the commented code will not get it to the branch.

File: CAR P-I/can_solve/can_solve.ino
Comment (2560p) [inline CAR P-I/can_solve/can_solve.ino:1]: LGTM, will be removed from the final version, so the commented code will not get it to the branch.

File: CAR P-I/final/final.ino
Comment (2560p) [inline CAR P-I/final/final.ino:1]: The final version, is to be merged.

File: CAR P-I/example.txt
Comment (2560p) [inline CAR P-I/example.txt:1]: Delete the empty file.

File: CAR P-I/distanceSensor/sketch_feb19a/sketch_feb19a.ino
Comment (2560p) [inline CAR P-I/distanceSensor/sketch_feb19a/sketch_feb19a.ino:1]: LGTM, will be removed from the final version, so the commented code will not get it to the branch.

File: CAR P-I/distanceSensor/distanceSensor.ino
Comment (2560p) [inline CAR P-I/distanceSensor/distanceSensor.ino:1]: LGTM, will be removed from the final version, so the commented code will not get it to the branch.

File: CAR P-I/wifi/wifi.ino
Comment (2560p) [inline CAR P-I/wifi/wifi.ino:1]: Did not make it to the final version due to the complexity and the deadline. Is to be removed.

**Review by 2560p — APPROVED** [review]
Approve! Need to make sure that the unused code will not make it to `main`

**Comment by 2560p** [comment]
The branch was merged!

Hooray!!!!!!!

**Pull request merged**